src/main/java/org/github/guppy/backend/controller/ProductController.java

```diff
@@ -0,0 +1,14 @@
+package org.github.guppy.backend.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ProductController {
+
+    @GetMapping("v1/products")
```

Comment:
Solo por cuestion de prolijidad
```suggestion
    @GetMapping("/v1/products")
```